### DISCUSSION TITLE
Feature: New options

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [savannabits/filament-modules#111] [Ideas]
Thank you for this amzaing package
i would recommend to add these features:
1. Add command for create panel in Module
2. When module is disabled the panel should be disabled  too

**Comment by coolsam726** [comment]
@mstfkhazaal you are welcome!

1. This is done in v5. See the beta release #137 
2. I hadn't thought about this, good enhancement. I will add it before the stable release.

Thank you for your contributions.
